src/main/java/dev/slimevr/protocol/DataFeedBuilder.java

```diff
@@ -192,7 +190,7 @@ public static int createDeviceData(
 		if (!mask.getDeviceData())
 			return 0;
 
-		if (device.getTrackers().size() <= 0)
+		if (device.getTrackers().size() == 0)
```

Comment:
Why? I don't see any reason to do `==`, `<=` seems fine to me, doesn't have any performace difference, and provides safety with implementations or extremely weird bugs